native/cocos/bindings/manual/jsb_conversions_spec.cpp

```diff
@@ -133,6 +134,17 @@ enum class DataType {
     INT,
     FLOAT
 };
+enum class MathType {
```

Comment:
Add a `NOTE` comment that MathType here should synchronize which in material.jsb.ts.

Reply:
I guess the note should be added in material.jsb.ts? that's more logical